tools/small_rna_map/small_rna_map.xml

```diff
@@ -0,0 +1,39 @@
+<tool id="small_rna_map" name="small rna map" version="0.0.1">
+  <description></description>
+  <requirements>
+        <requirement type="package" version="3.1.2">R</requirement>
+        <requirement type="package" version="1.3.0=r3.2.2_1">r-optparse</requirement>
```

Comment:
You also need to specify numpy in your requirements as in https://github.com/ARTbio/tools-artbio/blob/master/tools/lumpy/lumpy.xml#L6 and probably ggplot for the future (look at bioconda if ggplot package is available)